assets/css/modules-frontend.scss

```diff
@@ -1,5 +1,5 @@
 $fontawesome: FontAwesomeSensei, FontAwesome;
-.module-archive #main .status, #main .course .module-status {
```

Comment:
> Is this element completely removed?

It depends on the theme, but in general, yes.

> It seems as though it may be safer to leave it in, and then add new selectors that exclude `#main`.

Hmm. I see what you're getting at, but I don't think this would make any difference, technically. For example, the selector `.module-archive #main .status, .module-archive .status` will match the exact same elements as `.module-archive .status`.

Removing the `#main` selector effectively makes the rules less specific. I.e. they will still match all the elements that would have been matched previously, but they will match other elements as well.

So the potential problem here is not that elements will be missed, but that we will match elements that we don't intend to. The rules are all scoped to `.module-archive` or `.course`, which gives us some safety. But technically, the possibility exists. We might be able to make the rules more specific, even after removing `#main`. Do you think that's worth looking into?

Reply:
Nah, we'll likely uncover this during testing, if a problem exists.